draft-thomson-http-mice.md

```diff
@@ -273,6 +280,11 @@ which they were applied.
 The MI header MAY be omitted if the sender intends for the receiver to acquire
 the integrity proof for the first record by other means.
 
+*RFC EDITOR: Please remove the next paragraph before publication.*
+
+Implementations of drafts of this specification use the "MI" header field
```

Comment:
I left this so @nyaxt et. al. don't have to wonder if we just forgot this part, but I can also just tell them. Let me know if I should undo it.

Reply:
It's going anyway.